engine/class_modules/sc_death_knight.cpp

```diff
@@ -3939,14 +3943,14 @@ struct helchains_damage_t : public death_knight_spell_t
   }
 };
 
-struct helchains_buff_t : public buff_t
+struct unholy_pact_buff_t : public buff_t
```

Comment:
This seems to be missing the strength buff of Unholy Pact, using spell ID = 319255 (or talent.unholy_pact(319230) -> effectN( 1 ).trigger() (319232) -> effectN( 1 ).trigger() (319233) -> effectN( 4 ).trigger() (319255)